web/packages/teleterm/src/ui/Vnet/vnetContext.tsx

```diff
@@ -185,3 +189,29 @@ export const useVnetContext = () => {
 
   return context;
 };
+
+const notifyAboutDaemonBackgroundItem = async (ctx: IAppContext) => {
+  const { vnet, notificationsService } = ctx;
+
+  let backgroundItemStatus: BackgroundItemStatus;
+  try {
+    const { response } = await vnet.getBackgroundItemStatus({});
+    backgroundItemStatus = response.status;
+  } catch (error) {
+    // vnet.getBackgroundItemStatus returns UNIMPLEMENTED if tsh was compiled without the
+    // vnetdaemon build tag.
+    if (isTshdRpcError(error, 'UNIMPLEMENTED')) {
+      return;
+    }
+
+    throw error;
+  }
+
+  if (backgroundItemStatus === BackgroundItemStatus.ENABLED) {
+    return;
+  }
+
+  notificationsService.notifyInfo(
+    'Please enable the background item for tsh.app in the Login Items section of System Settings to start VNet.'
```

Comment:
Nit: I think it is easier to find in reverse order (and it's usually written this way in [help articles](https://support.apple.com/en-az/guide/deployment/depdca572563/web)).
```suggestion
    'Please enable the background item for tsh.app in System Settings > General > Login Items to start VNet.'
```

Reply:
Oh that's neat, thanks for bringing up the help articles.